config/locales/hyrax.en.yml

```diff
@@ -698,7 +698,7 @@ en:
           edit_access: 'Edit Access:'
           groups: 'Groups:'
           users: 'Users:'
-        collections: Your Collections
+        collections: Collections
```

Comment:
This translation is shared by breadcrumbs and tab label.  There needs to be a separate translation for each of these.  The breadcrumb is getting its label based on the title on the page seen when going to Dashboard -> Collections.  I suggest that you locate what drives the title of the page and have breadcrumbs share that label.  The alternative is to add another label specific to the tabs that has a value of Your Collections.

The same applies to Works around line 728.